src/Microsoft.Dnx.Tooling/Restore/EmbeddedRuntimeData.cs

```diff
@@ -72,7 +72,7 @@ internal static class EmbeddedRuntimeData
             new RuntimeSpec("ubuntu", "linux"),
             new RuntimeSpec("ubuntu-x64", "ubuntu", "linux-x64"),
 
-            new RuntimeSpec("ubuntu.14.04", "ubuntu"),
+            new RuntimeSpec("cd .ubuntu.14.04", "ubuntu"),
```

Comment:
Haha, and I totally know exactly how a typo like that happens. I've almost committed something with "jjj" in it from VsVim being in the wrong mode